lighthouse-core/audits/metrics/cumulative-layout-shift.js

```diff
@@ -44,8 +44,9 @@ class CumulativeLayoutShift extends Audit {
    */
   static get defaultOptions() {
     return {
-      // TODO(paulirish): Calibrate these
-      scorePODR: 0.05,
+      // Calibrated to assure 0.1 gets a score of 0.9.
```

Comment:
maybe a note in here or in the PR description or in the scoring doc on where these came from? As we've discovered it can be very difficult to figure out our motivation for some of these curves later on :)